.github/workflows/docs-ci.yml

```
@@ -191,3 +191,51 @@ jobs:
           remote_port: ${{ secrets.DOCS_PORT }}
           remote_user: ${{ secrets.DOCS_USER }}
           remote_key: ${{ secrets.DEEPHAVEN_CORE_SSH_KEY }}
+
+  rdoc:
+    runs-on: ubuntu-22.04
+    concurrency:
+      group: rdoc-${{ github.workflow }}-${{ github.ref }}
+      # We don't want to cancel in-progress jobs against main because that might leave the upload in a bad state.
+      cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup JDK 11
+        id: setup-java-11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Set JAVA_HOME
+        run: echo "JAVA_HOME=${{ steps.setup-java-11.outputs.path }}" >> $GITHUB_ENV
+
+      - name: Setup gradle properties
+        run: |
+          .github/scripts/gradle-properties.sh >> gradle.properties
+          cat gradle.properties
+
+      - name: Generate R Docs
+        uses: burrunan/gradle-cache-action@v1
+        with:
+          job-id: rDocs
+          arguments: R:rClientSite
+          gradle-version: wrapper
+
+      - name: Deploy R Docs
+        if: ${{ github.ref == 'refs/heads/main' }}
+        uses: burnett01/rsync-deployments@5.2
+        with:
+          switches: -avzr --delete
+          path: R/rdeephaven/docs/
+          remote_path: deephaven-core/client-api/R/
```

```suggestion
          remote_path: deephaven-core/client-api/r/
```

for consistency, I would lowercase the path please